README.md

```diff
@@ -1,90 +1,141 @@
+-   [Licence](#licence)
+
+# Documentation {#documentation}
+
+If you are reading this on GitHub README, please refer to the [online
+documentation](https://www.open-systems-pharmacology.org/OSPSuite-R/)
+for more details on the package.
+
+In particular, we would recommend that you read the articles in the
+following order:
+
+-   [Get
+    Started](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/ospsuite.html)
+-   [Loading a simulation and accessing
+    entities](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/load-get.html)
+-   [Changing parameter and molecule start
+    values](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/set-values.html)
+-   [Running a
+    simulation](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/run-simulation.html)
+-   [Efficient
+    calculations](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/efficient-calculations.html)
+-   [Creating
+    individuals](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/create-individual.html)
+-   [Population
+    simulations](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/create-run-population.html)
+-   [PK
+    Analysis](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/pk-analysis.html)
+-   [Sensitivity
+    analysis](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/sensitivity-analysis.html)
+-   [Table
+    parameters](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/table-parameters.html)
+-   [Dimensions and
+    Units](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/unit-conversion.html)
+-   [Working with data sets and import from
+    excel](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/observed-data.html)
+-   [Working with `DataCombined`
+    class](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/data-combined.html)
+-   [Visualizations with
+    `DataCombined`](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/data-combined-plotting.html)
+
+# Installation {#installation}
+
+The **ospsuite** package is compatible with R version 3.6.x **AND**
+4.x.x and can be used on [Windows](#on-windows) and [Linux](on-linux)
+operating systems.
+
+`ospsuite` depends on three packages that are not available on CRAN and
+need to be installed separately:
+
+-   `rClr`
+
+    -   [For R
+        4.x.x](https://github.com/Open-Systems-Pharmacology/rClr/releases/download/v0.9.2/rClr_0.9.2.zip)
+
+    -   [For R
+        3.6.x](https://github.com/Open-Systems-Pharmacology/rClr/releases/download/v0.9.1-R3/rClr_0.9.1.zip)
+
+-   [`ospsuite.utils`](https://github.com/Open-Systems-Pharmacology/OSPSuite.RUtils/releases/latest)
+
+-   [`tlf`](https://github.com/Open-Systems-Pharmacology/TLF-Library/releases/latest)
+
+## On Windows {#on-windows}
+
+### Pre-requisites
+
+The package requires the Visual C++ Runtime that is installed with OSPS
+and can be manually downloaded
```

Comment:
Another prereq is .NET Framework 4.7.2 which can be installed from https://go.microsoft.com/fwlink/?LinkID=863265
I would also mention that if the OSP Suite (in the version matching the version of the ospsuite-R) is already installed - there is no need to install those prereqs manually.